parentheses/parentheses.go

```diff
@@ -31,7 +31,28 @@ func IsBalanced(s string) bool {
 	return len(stack) == 0
 }
 
-func GenerateParantheses(w http.ResponseWriter, r *http.Request) {
+func GenerateSequence(length int) string {
+	var parentheses string
+	for i := 0; i < length; i++ {
+		switch rand.Perm(6)[0] {
```

Comment:
why are you using `Perm()`?

Reply:
switch rand.Perm(6) generates a slice containing a random permutation of the integers from 0 to 5. Using it helps string to be more unique.

Reply:
but, you're always only taking the first value in a permutation. How is it different from calling `rand.Intn(6)`?

Reply:
so should I change it with rand.Intn(6) or find different way?

Reply:
yes, I think so